packages/vuepress-theme-odyssey/styles/abstracts/_mixins.scss

```diff
@@ -0,0 +1,15 @@
+$breakpoints: (
```

Comment:
Looks like these haven't been updated to our layout-specific breakers?

Reply:
These can still change as part of a separate PR. @joejezowski-okta seems to be fine with how things are breaking down now, and content isn't final so I don't see a need to change them (right now)